bot.py

```diff
@@ -1,9 +1,11 @@
-import discord,re
-with open("disco_link.txt", "r") as f:
-    t = f.readline()
+import discord,os,responses
```

Comment:
where are these dependencies described as installable? I guess we have to use requirements.txt method.

Reply:
Got it, much appreciated!